highlight.io/pages/blog/[slug].tsx

```diff
@@ -62,6 +63,40 @@ const components = {
 	h3: (props) => <h6 className={styles.blogText}>{props.children}</h6>,
 	h4: (props) => <h6 className={styles.blogText}>{props.children}</h6>,
 	h5: (props) => <h6 className={styles.blogText}>{props.children}</h6>,
+	ul: (props) => {
```

Comment:
@deltaepsilon, I used your code [here](https://github.com/highlight/highlight/pull/5255) for this. I also made a change that makes sure that mutiline `li` elements are indented for each line.

Reply:
I'm not sure if we should re-use this in both places or if that would just complicate things. But at the very least, take a look at the multiline stuff and borrow it?